src/ui/CategoryFacet/CategoryFacetTemplates.ts

```diff
@@ -0,0 +1,42 @@
+import { $$, Dom } from '../../utils/Dom';
+import { SVGIcons } from '../../utils/SVGIcons';
+import { SVGDom } from '../../utils/SVGDom';
+import _ = require('underscore');
+
+export interface CategoryFacetData {
+  value: string;
+  count: number;
+}
+
+export class CategoryFacetTemplates {
```

Comment:
Not sure I like the name... It's a bit more than a template.

`CategoryValueRow` ?

Reply:
Well it's a helper class to build different templates needed to build a list of category values.

`CategoryFacetTemplatesBuilder`?

Reply:
I prefer the original name then :)